src/Workspaces/Remote/ServiceHub/Services/CodeAnalysisService_SymbolFinder.cs

```diff
@@ -31,6 +31,7 @@ internal partial class CodeAnalysisService : IRemoteSymbolFinder
                 }
 
                 var documents = documentArgs?.Select(solution.GetDocument)
+                                             .Where(d => d != null)
```

Comment:
I couldn't find `WhereNotNull`, but maybe it was due to the diagnostic issue Jason was looking at.  I'll try again.